<launch>
  <arg name="launch_voxblox" default="true" />

  <include file="$(find mesher_mapper)/launch/generate_mesh.launch" pass_all_args="true"/>

  <node type="kimera_rmpgo_node" name="kimera_rmpgo_node" pkg="mesher_mapper" output="screen" >
    <rosparam file="$(find mesher_mapper)/params/parameters.yaml" />

    <param name="output_ply_file" value="/home/yunchang/Downloads/optimized.ply" />

    <remap from="~input_mesh" to="/converted_mesh" />
    <remap from="~pose_graph_incremental" to="/kimera_vio_ros/pose_graph_incremental" />
  </node>

</launch>